docs/developer/soffits/minimal_soffit.md

```diff
@@ -0,0 +1,71 @@
+
+# Minimal Soffit
+
+A soffit's interaction with the portal is HTTP-based.  It is posible to write a
+soffit in any language or platform that can accept, process, and respond to a
+connection over HTTP.  At the present time, the creators of Soffit expect to
+develop soffits mostly with [Java][] and [Spring Boot][].
+
+## Minimal Soffit Setup Instructions Using Spring Boot
+
+1. Use the [Spring Initializer][] to create a new Spring Boot project with the
+   following settings:
+    * Gradle Project (recommended)
+    * Packaging=*War* (recommended)
+    * Dependencies=*Cache* (recommended) & *Web* (required)
+    * Additional dependencies you intend to use (optional -- you can add them
+      later)
+1. Add Soffit as a dependency to your project (see below)
```

Comment:
Write for readability. Use the style that includes the numbers in the documentation source.